# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/btrfs-progs/hotfix-big-endian.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- btrfs-progs-v6.3.1/kernel-shared/accessors.h.vanilla	2023-06-06 12:21:52.567471719 +0200
+++ btrfs-progs-v6.3.1/kernel-shared/accessors.h	2023-06-06 12:22:09.754473486 +0200
@@ -579,7 +579,7 @@
 BTRFS_SETGET_STACK_FUNCS(disk_key_offset, struct btrfs_disk_key, offset, 64);
 BTRFS_SETGET_STACK_FUNCS(disk_key_type, struct btrfs_disk_key, type, 8);
 
-#ifdef __LITTLE_ENDIAN
+#if __BYTE_ORDER == __LITTLE_ENDIAN
 
 /*
  * Optimized helpers for little-endian architectures where CPU and on-disk
